Server/validation/joi.js

```diff
@@ -144,10 +144,12 @@ const getMonitorsAndSummaryByTeamIdQueryValidation = joi.object({
 	type: joi
 		.alternatives()
 		.try(
-			joi.string().valid("http", "ping", "pagespeed", "docker", "hardware"),
+			joi.string().valid("http", "ping", "pagespeed", "docker", "hardware", "port"),
 			joi
 				.array()
-				.items(joi.string().valid("http", "ping", "pagespeed", "docker", "hardware"))
+				.items(
+					joi.string().valid("http", "ping", "pagespeed", "docker", "hardware", "port")
+				)
```

Comment:
We could extract these to a variable, maybe an array, and destructure them here. That way we can update in one place and use it in four, do you like the idea?

Reply:
Absolutely, it was a pain tracking these down all over the app 😂.

Good idea, how about I open a separate PR for that one?

Reply:
Yeah, it's better =)